akka-persistence-typed/src/test/scala/akka/persistence/typed/scaladsl/PrimitiveStateSpec.scala

```diff
@@ -43,7 +41,7 @@ class PrimitiveStateSpec extends ScalaTestWithActorTestKit(PrimitiveStateSpec.co
     }
 
   "A typed persistent actor with primitive state" must {
-    "persist events and update state" in {
+    "persist primitive events and update state" in {
```

Comment:
this test was already verifying primitive events (Int)

the original test from PersistentActorSpec was "be able to persist events that extend AnyVal" but it's not about AnyVal but primitive values so I renamed that